web/src/components/tags-input/index.tsx

```diff
@@ -80,8 +70,8 @@ export const TagsInput: React.FC<TagsInputProps> = props => {
       value,
       onKeyDown: e => {
         onKeyDown?.(e)
-        if (e.key === 'Backspace' && !value.length && tags && tags.length) {
-          onRemove(tags[tags.length - 1].id)
+        if (e.key === 'Backspace' && !value.length && children?.length) {
+          onRemove(children[children.length - 1].props.id)
```

Comment:
onRemove for TagsInput is never set? I get `Uncaught TypeError: onRemove is not a function` when I press backspace

Reply:
Yes, I added the `onRemove` function to the story. Thanks!